corehq/apps/users/bulkupload.py

```diff
@@ -487,12 +487,14 @@ def is_password(password):
                     if language:
                         user.language = language
                     if email:
-                        email_validator = EmailValidator()
                         try:
-                            email_validator(email)
-                            user.email = email
+                            validate_email(email)
                         except ValidationError:
                             raise UserUploadError(_("User has an invalid email address"))
+
+                        if not email.islower():
+                            raise UserUploadError(_("Email address must be lower case: {}".format(email)))
```

Comment:
Lol. That seems... silly. 
I wonder if forcing it to lowercase is a better option then? Our code did that until August last year:
https://github.com/dimagi/commcare-hq/commit/385ef4316c0093d10fe08cd9cbe39c17f79af49b 
And we do that for forms:
https://github.com/dimagi/commcare-hq/pull/7911/files